core/oputil/oputil_ssh_key.js

```diff
@@ -0,0 +1,158 @@
+/* jslint node: true */
+/* eslint-disable no-console */
+'use strict';
+
+//	ENiGMA½
+const initConfigAndDatabases = require('./oputil_common.js').initConfigAndDatabases;
+
+const {
+    printUsageAndSetExitCode,
+    argv,
+    ExitCodes,
+    getAnswers,
+} = require('./oputil_common.js');
+const getHelpFor = require('./oputil_help.js').getHelpFor;
+
+//	deps
+const async = require('async');
+const fs = require('graceful-fs');
+const exec = require('child_process').exec;
+const inq = require('inquirer');
+const _ = require('lodash');
+
+
+exports.handleSSHKeyCommand = handleSSHKeyCommand;
+
+const ConfigIncludeKeys = [
+    'loginServers.ssh',
+    'loginServers.ssh.privateKeyPem',
+];
+
+const MINIMUM_PASSWORD_LENGTH = 8;
+const QUESTIONS = {
+    Create: [
+        {
+            name: 'createNew',
+            message: 'Generate New SSH Keys?',
+            type: 'confirm',
+            default: false,
+        },
+        {
+            name: 'password',
+            message: 'SSL Password:',
+            default: "",
+            when: answers => answers.createNew,
+        },
+    ],
+};
+
+function execute(ui, command) {
+    ui.log.write("Ping!");
+    ui.log.write(command);
+    exec(
+        command,
+        function (error, stdout, stderr) {
+            ui.log.write(error);
+
+            if (error) {
+                const reason = error ? error.message : 'OpenSSL Error';
+                logDebug(
+                    {
+                        reason: reason,
+                        cmd: util.cmd,
+                        args: args
+                    },
+                    `openssl command failed`
+                );
+            }
+            else {
+                ui.log.write("SSH Keys Generated")
+            }
+        }
+    );
+}
+
+function createNew(cb) {
+    const ui = new inq.ui.BottomBar();
+
+    let sslPassword;
```

Comment:
this can just be assigned below, e.g. `const sslPassword = answers.password.trim()`

Reply:
Done